tableauserverclient/server/request_options.py

```diff
@@ -173,3 +173,11 @@ def apply_query_params(self, url):
         self._append_view_filters(params)
 
         return "{0}?{1}".format(url, '&'.join(params))
+
+
+def to_int(name, value):
+    try:
+        return int(value)
+    except ValueError:
+        print("Integer value expected for the '{}' parameter, but got {}".format(name, value))
```

Comment:
I don't think we want to just print something in this case.  We should throw an exception that tells what the error is. I believe we have something for Integer parameter errors

Reply:
We have property decorators in models for Integer parameter errors, but didn't want to bring that into here.
Above snippet prints a message and re-throws the same ValueError. It would be ideal to throw an exception with our custom message, but that results in 2 exceptions being thrown.